qlib/contrib/model/catboost_model.py

```diff
@@ -12,18 +15,32 @@
 from ...data.dataset.handler import DataHandlerLP
 from ...model.interpret.base import FeatureInt
 from ...data.dataset.weight import Reweighter
+from ...log import get_module_logger
+
+working_dir = os.getcwd()
+target_dir = os.path.join(os.path.relpath(working_dir), "tensor_board")
```

Comment:
I think we can avoid using it as a global variable.
Maybe local variable will be enough and better.